discovery/dns/dns_test.go

```diff
@@ -16,6 +16,7 @@ package dns
 import (
 	"context"
 	"fmt"
+	"gopkg.in/yaml.v2"
```

Comment:
could you move it the second group of imports (eg `github.com/go-kit/kit/log`)?

Reply:
ok.